test/unit/spec/EnrollU2F_spec.js

```diff
@@ -103,7 +113,46 @@ function (Okta,
     });
 
     Expect.describe('Enroll factor', function () {
+      itp('shows error if wrong browser', function () {
+        mockChrome(false);
+        mockFirefox(false);
+
+        return setup().then(function (test) {
+          expect(test.form.errorHtml()).toHaveLength(1);
```

Comment:
nit: might be a good idea to verify that the error message is the correct one, since we have different ones depending on the state
```
expect(test.form.errorHtml().indexOf('The Security Key is only supported for Chrome or Firefox browsers') > -1).toBe(true);
```